lib/src/svg-icon-registry.service.ts

```diff
@@ -1,22 +1,34 @@
-import { Injectable, Optional, SkipSelf } from '@angular/core';
+import { Inject, Injectable, InjectionToken, Optional, SkipSelf } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 
 import { Observable, of as observableOf, throwError as observableThrowError } from 'rxjs';
 import { map, tap, catchError, finalize, share } from 'rxjs/operators';
 
-@Injectable()
+import { PLATFORM_ID } from '@angular/core';
+import { DOCUMENT, isPlatformBrowser } from '@angular/common';
+
+export const SERVER_URL = new InjectionToken<string>('SERVER_URL');
+
+@Injectable({
+	providedIn: 'root',
```

Comment:
Is the `providedIn` still needed? I do not believe so since the service has a factory/provider instead.